R/geom-col.r

```diff
@@ -42,16 +44,19 @@ GeomCol <- ggproto("GeomCol", GeomRect,
     params
   },
 
-  extra_params = c("na.rm", "orientation"),
+  extra_params = c("just", "na.rm", "orientation"),
 
   setup_data = function(data, params) {
     data$flipped_aes <- params$flipped_aes
     data <- flip_data(data, params$flipped_aes)
     data$width <- data$width %||%
       params$width %||% (resolution(data$x, FALSE) * 0.9)
-    data <- transform(data,
-                      ymin = pmin(y, 0), ymax = pmax(y, 0),
-                      xmin = x - width / 2, xmax = x + width / 2, width = NULL
+    data$just <- data$just %||% params$just %||% 0.5
```

Comment:
Ah, sorry, I forgot to mention this `data$just` as well.

Reply:
Sorry for the confusion, but I think I was sane on this comment.

Reply:
Ah! Well spotted, it took me way to long to see what you were talking about there. I'll make the change.